angular2/src/angular2TextMask.ts

```diff
@@ -33,6 +33,7 @@ export default class MaskedInputDirective {
   }
 
   writeValue(value: any) {
+    this.textMaskInputElement.update(value)
```

Comment:
With this line, with every time the end-user enters a character, `textMaskInputElement.update` will execute twice. Is there a way to avoid this double execution?